api/v1beta1/cinderapi_types.go

```diff
@@ -84,6 +84,11 @@ type CinderAPISpec struct {
 	// +kubebuilder:validation:Required
 	// ServiceAccount - service account name used internally to provide Cinder services the default SA name
 	ServiceAccount string `json:"serviceAccount"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=memcached
+	// Memcached instance name.
+	MemcachedInstance string `json:"memcachedInstance"`
```

Comment:
I think this should be a `*string` to make sure we gracefully handle minor updates.
When is a *string, if this parameter doesn't exist, you won't trigger any Pod restart/rollout (you should just need to handle the code you already have with a `instance.Spec.MemcachedInstance != nil`.
By keeping the code as it is we risk to trigger a cinder restart when the operator is updated.